tools/src/tensil/tools/compiler/MemoryManager.scala

```diff
@@ -7,38 +7,34 @@ import java.io._
 import scala.collection.mutable
 import org.tensorflow.framework.types.DataType
 
-import _root_.tensil.tools.data.{Shape, TensorData}
-import _root_.tensil.tools.util
-import _root_.tensil.tools.{
-  CompilerException,
-  TraceContext,
-  TracepointCondition
+import tensil.tools.data.{Shape, TensorData}
```

Comment:
nice to see this weird `_root_` stuff go away

Reply:
Yeah, looks like they fixed Metals, so no more `_root_` stuff!